sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Added a WinHTTP-based `HttpTransport` called `WinHttpTransport` and use that as the default `TransportPolicyOptions.Transport` on Windows when sending and receiving requests and responses over the wire.
 - Added `Range` type to `Azure::Core::Http` namespace. 
+- Added support for long-running operations with `Operation<T>`.
```

Comment:
nit: fully qualify the type
```suggestion
- Added support for long-running operations with `Azure::Core::Operation<T>`.
```

Reply:
I don't think we should do it, it is less readable. Unless we move things from namespace to namespace.